src/ol/interaction/MouseWheelZoom.js

```diff
@@ -155,7 +155,11 @@ class MouseWheelZoom extends Interaction {
    */
   endInteraction_() {
     this.trackpadTimeoutId_ = undefined;
-    const view = this.getMap().getView();
+    const map = this.getMap();
+    if (!map) {
+      return;
+    }
+    const view = map.getView();
```

Comment:
This method is called in a timeout, with no guarantees that the interaction still has a map when it runs.  In the tests, we now remove interactions from the map in test cleanup, and this method was being called after the tests complete.